build/dist

```diff
@@ -245,7 +245,7 @@ if [[ "$MAKE_TGZ" == "true" ]]; then
   TARDIR="$KYUUBI_HOME/$TARDIR_NAME"
   rm -rf "$TARDIR"
   cp -r "$DISTDIR" "$TARDIR"
-  tar czf "$TARDIR_NAME.tgz" -C "$KYUUBI_HOME" "$TARDIR_NAME"
+  tar zcf "$TARDIR_NAME.tgz" -C "$KYUUBI_HOME" "$TARDIR_NAME"
```

Comment:
What's the difference?

Reply:
I am looking into it, and will test it locally.

Reply:
Does the different behavior also affected by BSD and GNU? Because macOS use BSD tar but Linux use GNU tar.

Reply:
maybe, I tried the tar command manually and it looks fine.